### DISCUSSION TITLE
Request Client Hints and have `next/image` respect the DPR value.

### DISCUSSION THREAD

**Discussion by Jared-Dev** [vercel/next.js#19221] [Ideas]
It would be awesome if a Next.JS site would request [Client Hints](https://developer.mozilla.org/en-US/docs/Glossary/Client_hints). Specifically, If we can get the DPR the `Image` component can intelligently request 2x/etc images. If a client only has a DPR of "1" then there is little reason for the 2x image to be included.

**Comment by cotwitch** [comment]
Also looking for this.
